### PR TITLE
Document how to provide container registry credentials in air-gapped environment

### PR DESCRIPTION
**Pull request by otteryoudoing**
Adding information on how to use credentials with a custom repo.

Added details to initial custom repo example.

Also provided a link later on to another example in Github.

### CODE REVIEW

File: docs/operating-eck/air-gapped.asciidoc [hunk 1]
```diff
@@ -53,6 +66,8 @@ The operator expects container images to be located at specific repositories in
 * +my.registry/kibana/kibana:{version}+
 * +my.registry/apm/apm-server:{version}+
 
+You can provide credentials to a custom repo by adding the imagePullSecrets parameter to your configuration as shown here - https://github.com/elastic/cloud-on-k8s/issues/2580#issuecomment-589602837
```
Comment (thbkrkr) [inline docs/operating-eck/air-gapped.asciidoc:69]:
```suggestion
You can provide credentials to your private container registry by setting the `imagePullSecrets` field through the `spec.podTemplate` section of your Elastic resource specification, check <<{p}-customize-pods,how to customize the Elastic resources Pods>>.
```

File: docs/operating-eck/air-gapped.asciidoc [hunk 2]
```diff
@@ -53,6 +56,7 @@ The operator expects container images to be located at specific repositories in
 * +my.registry/kibana/kibana:{version}+
 * +my.registry/apm/apm-server:{version}+
 
+
 [float]
```
Comment (thbkrkr) [inline docs/operating-eck/air-gapped.asciidoc:60]:
```suggestion
[float]
```

File: docs/operating-eck/air-gapped.asciidoc (hunk 1 above)
Comment (thbkrkr) [reply docs/operating-eck/air-gapped.asciidoc]: Due to lack of activity, I applied the suggestion. I hope it is okay for you.

**Review by naemono — APPROVED** [review]
This looks good.

**Review by barkbay — APPROVED** [review]
LGTM

**Pull request merged**